projects/aws/upgrader/Help.mk

```diff
@@ -0,0 +1,64 @@
+
+
+
```

Comment:
Is this intentional?

Reply:
this file should be auto generated, if we have to make manual changes, lets fix the script instead.

Reply:
this is auto-generated. not sure if its intended this way, but i see same 3 new lines in all other sub -projects under AWS